### PR TITLE
fix(actionsheet): style and behaviour

### LINKED ISSUE #5427: Mobile - fix action sheet style/behaviour

### LINKED ISSUE #5428: iOS - fix context menu behaviour

### PR DESCRIPTION
**Pull request by jasonwoodland**
### What this PR does 📖
- Fix viewport meta tag (by disabling meta on nuxt bridge config)
- Set user-scalable to no – fixes triple tapping zooming in on some elements on iOS
- Disable dragging anchor tags on iOS
- Fix iOS long touch events for context menu
- Fix action sheet style and animation

### Which issue(s) this PR fixes 🔨
- Resolve #5427 
- Resolve #5428

### Special notes for reviewers 🗒️
- 

### Additional comments 🎤
-

### CODE REVIEW

File: assets/styles/base.less [hunk 1]
```diff
@@ -40,8 +40,6 @@ html {
 }
 
 body {
-  padding: env(safe-area-inset-top) env(safe-area-inset-right)
-    env(safe-area-inset-bottom) env(safe-area-inset-left) !important;
```
Comment (molimauro) [inline assets/styles/base.less:44]: Are these not needed anymore in mobile? :)

Comment (jasonwoodland) [reply assets/styles/base.less:44]: No, we're doing it in the layout components now using the new `var(--safe-area-inset-*)` variables 😎

Comment (molimauro) [reply assets/styles/base.less:44]: Dope thanks 🚀

**Review by nathan-power — CHANGES_REQUESTED** [review]
Looking good, just a little issue when opening the action-sheet on the friends screen (found in browser)
スクリーンショット 2022-10-31 10 55 11

**Comment by jasonwoodland** [comment]
@nathan-power should be fixed

**Review by molimauro — APPROVED** [review]

**Comment by jasonwoodland** [comment]
@phillsatellite 
I fixed some of these, still have to tackle the action sheet being hidden outside of the container. But I hope to fix this in a future PR so we can avoid conflicts. 😊

**Pull request merged**